yarn.lock

```diff
@@ -15,9 +15,9 @@
   integrity sha512-np/lG3uARFybkoHokJUmf1QfEvRVCPbmQeUQpKow5cQ3xWrV9i3rUHodKDJPQfTVX61qKi+UdYk8kik84n7XOw==
 
 "@babel/highlight@^7.10.4":
-  version "7.13.8"
```

Comment:
Were these changes accidental? I don't see any dependency changes in this PR

Reply:
Yeah, that was accidental. I think this got rid of lockfile changes: f8ff174